test/e2e/gymnast.spec.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable global-require, import/no-dynamic-require */
+import * as Gymnast from 'gymnast'
+import asGrid from 'gymnast/asGrid'
+import asLayout from 'gymnast/asLayout'
+import Col from 'gymnast/col'
+import ConfigProvider from 'gymnast/configProvider'
+import defaults from 'gymnast/defaults'
+import Dev from 'gymnast/dev'
+import Grid from 'gymnast/grid'
+import Layout from 'gymnast/layout'
+import log from 'gymnast/log'
+import Offset from 'gymnast/offset'
+import Root from 'gymnast/root'
+import utils from 'gymnast/utils'
+import withResolution from 'gymnast/withResolution'
+
+function lowercaseFirstLetter(str = '') {
+  return str[0].toLowerCase() + str.slice(1)
+}
+
+const imports = [
+  'asGrid',
+  'asLayout',
+  'Col',
+  'ConfigProvider',
+  'defaults',
+  'Dev',
+  'Grid',
+  'Layout',
+  'log',
+  'Offset',
+  'Root',
+  'utils',
+  'withResolution',
+]
+
+describe('e2e tests', () => {
+  it('should export the same values from the main export than individual files', () => {
```

Comment:
What if this were just a snapshot test?

Reply:
This validates it's really the same export, with the snapshot test we would only validate that the export is of the same type. It's probably fine either way, this seemed more thorough but with the downside of having to manually enforce consistency. I added the test after that to make sure a test fails if an export is added and not included here.

Do you feel like it would be better with snapshot tests?

Reply:
my thought for a snapshot is that if we add an API a snapshot test would help remind us to add it to the test automatically, rather than needing to remember to manually update this definition

Reply:
that's fair, ok I'll make the update